src/Movie/Movie.css

```diff
@@ -29,6 +29,7 @@
 .movie-description-container {
     display: flex;
     margin: 10vh 0;
+    height: 50%;
```

Comment:
Gave height to the container to fix a sizing issue with the movie poster. The poster was getting squashed when the screen was narrow.